CHANGELOG.md

```diff
@@ -1635,6 +1635,7 @@ or in this [tutorial](http://docs.timescale.com/tutorials/continuous-aggs-tutori
 * #1177 Add transformed time_bucket comparison to quals
 * #1182 Enable optimizing SELECTs within INSERTs
 * #1201 Add telemetry for policies: drop_chunk & reorder
+* #4335 Show warnings when not following best practices
```

Comment:
This is in wrong place you should add your bugfix at the top of the file to the next version not to version 1.3.0